### PR TITLE
Bug fixes

### LINKED ISSUE #1: Fixed value 183891
Hi, it is not clear for me why in the getVecs function,  
`if(vecs_type == 'Test'):
    index = index + 183891`
Is there any reason to put that value?

### PR DESCRIPTION
**Pull request by avinashsai**
#1 Use Infer_vectors:
   Doc2Vec model should only be trained using training data and infer_vectors should be used for test 
  data.Hence modified the model to use only training data.
#2 random.shuffle()
  This function shuffles the data given to doc2vec model in every epoch and hence the sentiment labels 
  will not map to their corresponding vectors. Hence modified get_vectors() function to map labels with 
  their  corresponding vectors.
#3 gensim version updates:
    changed epochs=d2v.iter to epochs=epochs=d2v.epochs according to latest gensim version

### CODE REVIEW

**Comment by ibrahimsharaf** [comment]
Hi @avinashsai, thanks for the PR!

Let's split this PR into 3 separate new PRs, that would be better for readability, reviewing and testing.

- 1st one for updating `requirements.txt` file to the latest versions of needed packages, with needed syntax changes? 
- 2nd and 3rd PRs for the first 2 mentioned bugs.

Thanks again for collaborating :)

**Comment by ibrahimsharaf** [comment]
For the first PR (updating requirements), what do you think about using https://pyup.io/ for dependency updates? I think it's quite cool.

**Comment by avinashsai** [comment]
Yeah, it's really good. Probably we can consider using it.

**Comment by avinashsai** [comment]
@ibrahimsharaf What is the next step to proceed further?
